.github/workflows/build-test-linux.yml

```diff
@@ -20,24 +20,11 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build:
-    name: Build
-    runs-on: ${{ matrix.os }}
+  build-linux:
```

Comment:
Please change this to ```linux-unittest``` for consistency with [build-and-test-window](https://github.com/aws/amazon-cloudwatch-agent/blob/master/.github/workflows/build-test-windows.yml#L22)